src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -80,11 +82,30 @@ LoadFeeTrack::lowerLocalFee ()
 
 //------------------------------------------------------------------------------
 
-// Scale from fee units to millionths of a ripple
-std::uint64_t
-scaleFeeBase(std::uint64_t fee, Fees const& fees)
+template <class T1, class T2,
+    class = std::enable_if_t <
+        std::is_integral<T1>::value &&
+        std::is_unsigned<T1>::value &&
+        sizeof(T1) <= sizeof(std::uint64_t) >,
```

Comment:
I'm good with that.  And I'm also not as worried about the template definition since it's tucked away in a *.cpp file.  Hmmm.  Does it make sense to tag `lowestTerms()` as `static` since it's only used in this file?